plugins/module_utils/network/iosxr/config/interfaces/interfaces.py

```diff
@@ -166,6 +167,7 @@ def _state_replaced(self, want, have):
         commands = []
 
         for interface in want:
+            interface["name"] = normalize_interface(interface["name"])
```

Comment:
Looks like we're running this at multiple places. Would it make more sense to do this just in set_config() once?

Reply:
ack,will add it